docs/readthedocs/source/doc/Orca/Tutorial/yarn.md

```diff
@@ -298,7 +301,7 @@ If you prefer to use `spark-submit` instead of `bigdl-submit`, please follow the
     ```bash
     pip install -r /path/to/requirements.txt
     ```
-    Note that you are recommended **NOT** to install BigDL Orca in the conda environment if you use spark-submit to avoid possible conflicts.
+    Note that you are recommended **NOT** to pip install BigDL Orca in the conda environment if you use spark-submit to avoid possible conflicts.
```

Comment:
maybe better to `install BigDL Orca in the conda environment with the pip install command`?